src/main/archetype/ui.config/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
all contained OSGi configuration files should rather be in format `.config` or even `cfg.json` for the reasons outlined at https://sling.apache.org/documentation/bundles/configuration-installer-factory.html.